error-handler.js

```diff
@@ -258,9 +258,17 @@ createHandler = function createHandler(options) {
         }
       },
 
+      shouldContinue = function
```

Comment:
Holding state? What state are you referring to?
This seems to be quite a contradiction from your previous statements.
Please choose one:
1. Don't expose the convenience method status() back to the user. I realize its just the default method that reads environment variables, OR the user override, but exposing this single method to the user is not useful to a 503 handler like you've shown.
2. I've rethought this, and this is not in the interest of express-error-handler to read environment variables. Make your own version please.